docs/applications/pyserver/installation.rst

```diff
@@ -1,60 +1,6 @@
 Installation
 ---------------
 
-It is required that you have python's `pip` installed.
```

Comment:
I wouldn't delete all this! It provides more info than the dev-setup tutorial 
(especially the non-dev setup with pip)